components/member_list_team/index.ts

```diff
@@ -2,22 +2,35 @@
 // See LICENSE.txt for license information.
 
 import {connect} from 'react-redux';
-import {bindActionCreators} from 'redux';
+import {bindActionCreators, Dispatch} from 'redux';
 
 import {getTeamStats} from 'mattermost-redux/actions/teams';
 import {haveITeamPermission} from 'mattermost-redux/selectors/entities/roles';
 import {getMembersInCurrentTeam, getCurrentTeamStats} from 'mattermost-redux/selectors/entities/teams';
 import {getProfilesInCurrentTeam, searchProfilesInCurrentTeam} from 'mattermost-redux/selectors/entities/users';
 import {Permissions} from 'mattermost-redux/constants';
 import {searchProfiles} from 'mattermost-redux/actions/users';
+import {GlobalState} from 'mattermost-redux/types/store';
 
 import {loadStatusesForProfilesList} from 'actions/status_actions.jsx';
 import {loadProfilesAndTeamMembers, loadTeamMembersForProfilesList} from 'actions/user_actions.jsx';
 import {setModalSearchTerm} from 'actions/views/search';
 
-import MemberListTeam from './member_list_team.jsx';
+import MemberListTeam from './member_list_team';
 
-function mapStateToProps(state, ownProps) {
+interface OwnState extends GlobalState {
+    views: {
+        search: {
+            modalSearch: string;
+        };
+    };
+}
+
+type OwnProps = {
```

Comment:
Would be nice to have consistent naming convention but will defer to other Devs if we do have such style guide to follow. In other component, we're simply calling this `Props/State`.

Reply:
I have done this to somehow separate the actual props of the component with the props used on index. If you prefer to use also `Props/State` I am more than willing to change it.

Reply:
I'm 1/5 that it should consistently use `Props/State` (code quality) unless there is more than one similar items to name in a file (like if there is more than one components).  But I'm also hesitant to request for a change if there's no style guide to follow.  I'll defer to @devinbinnie being one of the owners of this campaign.

Reply:
I agree with following the naming conventions of other files